modules/installation-gcp-user-infra-init-bootstrap.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_gcps_user_infra/installing-gcp-user-infra.adoc
+
+[id="installation-gcp-user-init-infra-bootstrap_{context}"]
+= Initializing the bootstrap machine on GCP with user-provisioned infrastructure
+
+After you create all of the required infrastructure in Google Cloud Platform (GCP),
+you can install the cluster.
```

Comment:
This module isn't about installing the cluster; see @kalexand-rh's #16790.  Looks like a similar rewording is needed in `modules/installation-aws-user-infra-bootstrap.adoc` too.